CONTRIBUTING.md

```diff
@@ -35,22 +35,54 @@ Contributions are essential for keeping this extension great. We try to keep it
 	```bash
 	$ mvnw.cmd verify
 	```
-       
+
 
 5. `cd vscode-xml/`
-   
-6. Install the dependencies:  
-	```bash  
+
+6. Install the dependencies:
+	```bash
 	$ npm install
 	```
 
 7. In `vscode-xml/`, build the server by running:
 
-	```bash   
+	```bash
 	$ npm run build-server
 	```
 
 8. To run the extension, open the Debugging tab in VSCode.
 9. Select and run 'Launch Extension (vscode-xml)' at the top left:
 
     ![ Launch Extension ](./images/LaunchExtension.png)
+
+## Binary server development
+
+### Testing a binary version of LemMinX
+
+1. Copy the binary version of LemMinX to:
```

Comment:
Is it worth mentioning `xml.server.binary.path` as an alternative or should we just keep it simple? In a lot of testing I actually generate a vsix which I then install (as opposed to launching a child instance) so the path setting has been useful here.

Reply:
I'll mention it.